src/pykeen/pipeline/api.py

```diff
@@ -658,20 +660,23 @@ def pipeline(  # noqa: C901
     optimizer: HintType[Optimizer] = None,
     optimizer_kwargs: Optional[Mapping[str, Any]] = None,
     clear_optimizer: bool = True,
-    # 6. Training Loop
+    # 6. Learning Rate Scheduler
```

Comment:
oh no this numbering was in so many places. Maybe just call it 6B to make the diff smaller?

Reply:
What about 6.1?

Reply:
what about dropping these numbers?

```
# Learning Rate Scheduler
```

is just as descriptive as

```
# 6. Learning Rate Scheduler
```

Reply:
Why do we have them anyway?

Reply:
The numbers are there because some of the functions have 40+ kwargs and this helps navigate/organize them a bit better

Reply:
Now it became 5.1. Do we stick with this then?